src/topology/algebra/matrix.lean

```diff
@@ -0,0 +1,51 @@
+/-
+Copyright (c) 2021 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import linear_algebra.determinant
+import topology.algebra.ring
+
+/-!
+# Topological properties of matrices
+
+This file is a place to collect topological results about matrices.
+
+## Main definitions:
+
+ * `continuous_det`: the determinant is continuous over a topological ring.
+-/
+
+open matrix
+
+variables {ι k : Type*} [topological_space k]
+
+instance : topological_space (matrix ι ι k) := Pi.topological_space
+
+variables [fintype ι] [decidable_eq ι] [comm_ring k] [topological_ring k]
+
+lemma continuous_det : continuous (det : matrix ι ι k → k) :=
+begin
+  suffices : ∀ (n : ℕ), continuous (det : matrix (fin n) (fin n) k → k),
```

Comment:
If you change this to
```suggestion
  suffices : ∀ (n : ℕ), continuous (λ A : matrix (fin n) (fin n) k, matrix.det A),
```
and change the `rw coe_det_is_empty` to a `simp_rw`, you can get rid of the `change` line later, line 41.

Reply:
Should we have this lemma:
```lean
lemma continuous_delta {α β : Type*} [topological_space α] [topological_space β] (f : α → β) :
  continuous f ↔ continuous (λ x, f x) := iff.rfl
```
so that `change` isn't needed?

Reply:
Well, what I'm saying is that if you `have` something and then you have to `change` it later, it's better just to `have` to the thing you wanted in the first place.

Reply:
I agree with you both here. In this case I'm using `have` anyway so I should certainly `have` what I need but I think `continuous_delta` is worth having for other cases.